### PR TITLE
fix: _notify and _query kwargs, params and UI step parameters/action params..

### LINKED ISSUE #1179: Provider Parameters in workflow builder are missing for providers with both step and action
**Describe the bug**
For providers which provide both `_query` and `_notify` definitions in backend, the Provider Parameters in workflow builder show the same step parameters for both provider-step and provider-action. 

Here, the `stepParams` gets set to `properties.stepParams` regardless of the type of the item. 
https://github.com/keephq/keep/blob/575ce23ecf4c84a6ff331f85dc094732b55f3344/keep-ui/app/workflows/builder/editors.tsx#L62C1-L64C21

If the type is a step, then it should be `properties.stepParams`, otherwise `properties.actionParams`.

**To Reproduce**
1. Go to https://platform.keephq.dev/workflows
2. Click on "Create a Workflow"
3. Add any provider from toolbox which has both `step` and `action`, for example - `ilert-step` and `ilert-action`.
4. You'll see that the `Provider Parameters` in the Editor are same, whereas the `_notify` definition is - 
`
def _notify(
        self,
        summary: str = "",
        status: IlertIncidentStatus = IlertIncidentStatus.INVESTIGATING,
        message: str = "",
        affectedServices: str | list = "[]",
        id: str = "0",
        **kwargs: dict,
    ):
`
So, the`ilert-action` should have the parameters as `summary`, `status`, `message`, etc..

**Expected behavior**
The provider-action items should have the `properties.actionParams` and provider-step should have `properties.stepParams`.

**Screenshots**

https://github.com/keephq/keep/assets/56185464/0006fa24-cfdb-4244-832b-bb7e71bc8242

^^ Same parameters for step and action of ilert provider, not correct.

**Additional context**
Possible fix - 
If the type is a step, then it should be `properties.stepParams`, otherwise `properties.actionParams`.

After fix it should look like - 

https://github.com/keephq/keep/assets/56185464/937585f8-63f2-4b45-8c24-05ab2b53bb43

If the fix looks fine, I'll create a pull request.

### PR DESCRIPTION
**Pull request by KanvaBhatia**
Closes #1179

## 📑 Description

1. Fixed the parameters for `_notify` and `_query` for the providers which were using `**kwargs`.
2. Fixed UI for the workflow builder page, video attached here - #1179 

1. AxiomProvider - 
    1. _query - 
        1. dataset
2. BashProvider - 
    1. _query - 
        1. timeout, command
3. CloudwatchProvider -
    1.  _query - 
        1. log_group, query, hours
4. ConsoleProvider -
    1. _notify -
        1. message
5. GithubWorkflowsProvider -
    1. changed notify -> _notify, removed usage of kwargs to parameters -
        1. github_url
        2. github_method
6. GitlabpipelinsProvider - 
    1. fixed spelling of it’s AuthConfig class, which was causing no parameters to appear in UI.
    2. changed notify -> _notify, moved kwargs to parameters -
        1. github_url
        2. github_method
7. GoogleChatProvider -
    1. changed notify -> _notify
8. HttpProvider - 
    1. changed notify -> _notify
9. PlannerProvider - 
    1. changed notify -> _notify
10. OpsgenieProvider - 
    1. changed notify -> _notify
11. PagerdutyProvider - 
    1. changed notify -> _notify and fixed returning values properly
12. PagertreeProvider - 
    1. changed notify -> _notify
13. PostgresProvider - 
    1. changed notify -> _notify, added params instead of kwargs
        1. query
    2. _query - added params instead of kwargs
        1. query
14. PythonProvider - 
    1. _query -> added params
        1. code
        2. imports
15. TelegramProvider - 
    1. _notify -> added params
        1. chat_id
        2. message
16. TrelloProvider - 
    1. _query -> added params
        1. board_id
        2. filter
17. TwilioProvider - 
    1. _notify -> added params - 
        1. message_body
        2. to_phone_number
18. ZendutyProvider - 
    1. _notify -> added params - 
        1. title
        2. summary
        3. service
        4. user
        5. policy

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

File: keep-ui/app/workflows/builder/editors.tsx [hunk 1]
```diff
@@ -51,17 +51,17 @@ interface keepEditorProps {
   updateProperty: (key: string, value: any) => void;
   installedProviders?: Provider[] | null | undefined;
   providerType?: string;
+  type?: string;
```
Comment (talboren) [inline keep-ui/app/workflows/builder/editors.tsx:54]: @KanvaBhatia where does this come from? who providers it?

**Review by talboren — COMMENTED** [review]
1 small comment, overall looks great -- thank you so much for it!

File: keep-ui/app/workflows/builder/editors.tsx (hunk 1 above)
Comment (KanvaBhatia) [reply keep-ui/app/workflows/builder/editors.tsx:54]: I used the `type` here - 
https://github.com/keephq/keep/blob/143c8eb48db5ec291e53d8981a4e1003344c663f/keep-ui/app/workflows/builder/editors.tsx#L398C2-L416C11

It is coming from here - 
https://github.com/keephq/keep/blob/143c8eb48db5ec291e53d8981a4e1003344c663f/keep-ui/app/workflows/builder/editors.tsx#L391C3-L392C21

It's coming from `useStepEditor()` of `sequential-workflow-designer-react`

File: keep-ui/app/workflows/builder/editors.tsx [hunk 2]
```diff
@@ -413,6 +413,7 @@ export default function StepEditor({
           updateProperty={setProperty}
           installedProviders={installedProviders}
           providerType={providerType}
+          type={type}
```
Comment (KanvaBhatia) [inline keep-ui/app/workflows/builder/editors.tsx:416]: @talboren here it is

**Review by talboren — APPROVED** [review]
LGTM!

**Pull request merged**